<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FTP客户端</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../heart.ico</normaloff>../../heart.ico</iconset>
  </property>
  <property name="toolTipDuration">
   <number>-1</number>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_ftp">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>FTP服务器地址:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ftp">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>400</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Arial&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_user">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>用户名：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_user">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Arial&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_password">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>100</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>密码：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_password">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>100</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>160</y>
      <width>450</width>
      <height>30</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_upload">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>上传</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_download">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>220</y>
      <width>180</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>下载</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>580</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
